config/management/genesis/src/command.rs

```diff
@@ -213,7 +223,23 @@ pub mod tests {
             .set_layout(temppath.path().to_str().unwrap())
             .unwrap();
 
-        // Step 2) Upload the root keys:
+        // Step 2) Upload the Move modules
+        let tempdir = diem_temppath::TempPath::new();
+        tempdir.create_as_dir().unwrap();
+        for b in diem_framework_releases::current_module_blobs() {
+            let mut temppath =
+                diem_temppath::TempPath::new_with_temp_dir(tempdir.path().to_path_buf());
+            temppath.create_as_file().unwrap();
+            temppath.persist(); // otherwise, file will disappear before we call set_move_modules
```

Comment:
I imagine this gets cleaned up by the tempdir?

Reply:
Correct.